src/store/command/property-value-command.ts

```diff
@@ -0,0 +1,166 @@
+import { Command } from './command';
+import { Page } from '../page/page';
+import { PageElement } from '../page/page-element';
+import { Store } from '../store';
+
+/**
+ * A user operation to set the value of a page element property.
+ */
+export class PropertyValueCommand extends Command {
```

Comment:
As discussed, move the discussion wether a `PropertyValueCommand` is sealed here instead of the global store